src/containers/Rubric/index.test.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import selectors from 'data/selectors';
+import { Rubric, mapStateToProps } from '.';
+
+jest.mock('containers/CriterionContainer', () => 'CriterionContainer');
+jest.mock('./RubricFeedback', () => 'RubricFeedback');
+
+jest.mock('@edx/paragon', () => {
+  const Card = () => 'Card';
+  Card.Body = () => 'Card.Body';
+  const Button = () => 'Button';
+  return { Button, Card };
+});
+
+jest.mock('data/selectors', () => ({
+  __esModule: true,
+  default: {
+    app: {
+      isGrading: jest.fn((...args) => ({ isGragrding: args })),
+      rubric: {
+        criteriaIndices: jest.fn((...args) => ({
+          _rubricCriteriaIndices: args,
+        })),
+      },
+    },
+  },
+}));
+
+describe('Rubric Container', () => {
+  const props = {
+    isGrading: true,
+    criteriaIndices: [1, 2, 3, 4, 5],
+  };
+  describe('snapshot', () => {
+    test('is grading', () => {
+      expect(shallow(<Rubric {...props} />)).toMatchSnapshot();
+    });
+    test('is not grading', () => {
+      expect(
+        shallow(<Rubric {...props} isGrading={false} />),
+      ).toMatchSnapshot();
+    });
+  });
+
+  describe('component', () => {
+    test('is grading', () => {
+      const el = shallow(<Rubric {...props} />);
+      expect(el.find('.grading-rubric-footer').length).toEqual(1);
+    });
+
+    test('is not grading', () => {
```

Comment:
this is a description of the case, but not what you are testing.
```suggestion
    test('is not grading (no footer)', () => {
```